### PR TITLE
[android_intent_plus] nullsafety migration

### PR DESCRIPTION
**Pull request by miquelbeltran**

### CODE REVIEW

**Comment by miquelbeltran** [comment]
`0.5.0-nullsafety.1` published

File: packages/android_intent_plus/example/pubspec.yaml [hunk 1]
```diff
@@ -11,7 +11,7 @@ dependencies:
     path: ../
 
 dev_dependencies:
-  e2e: ^0.3.0
+  integration_test: ^1.0.2+2
```
Comment (miquelbeltran) [inline packages/android_intent_plus/example/pubspec.yaml:14]: e2e is deprecated and replaced by integration_test.

Unfortunately `integration_test` does not support null safety yet!

File: packages/android_intent_plus/example/test_driver/android_intent_plus_e2e.dart [hunk 2]
```diff
@@ -70,7 +70,7 @@ void main() {
     final intent = AndroidIntent(
       action: 'action_view',
       package: 'io.flutter.plugins.androidintentexample',
-      componentName: 'io.flutter.embedding.android.FlutterActivity',
+      componentName: 'io.flutter.plugins.androidintentexample.MainActivity',
```
Comment (miquelbeltran) [inline packages/android_intent_plus/example/test_driver/android_intent_plus_e2e.dart:73]: this was a bug unrelated with the null safety part, but necessary to pass the tests.

File: packages/android_intent_plus/CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,7 @@
+## 0.5.0-nullsafety.1
+
+- Null safafety support.
```
Comment (jpnurmi) [inline packages/android_intent_plus/CHANGELOG.md:3]: typo

File: packages/android_intent_plus/pubspec.yaml [hunk 4]
```diff
@@ -14,15 +14,18 @@ flutter:
 dependencies:
   flutter:
     sdk: flutter
-  platform: '>=2.0.0 <=3.0.0'
-  meta: ^1.0.5
+
+  platform: ^3.0.0-nullsafety.4
+  meta: ^1.3.0-nullsafety.6
```
Comment (jpnurmi) [inline packages/android_intent_plus/pubspec.yaml:19]: the migration guidelines recommend leaving out `.x` suffixes to make version solving more flexible

**Review by jpnurmi — APPROVED** [review]
LGTM 👍

**Comment by miquelbeltran** [comment]
thanks for checking @jpnurmi !

**Comment by jpnurmi** [comment]
@miquelbeltran np, congrats for the first null-safe package! 🎉

**Pull request merged**